Issue resolutions from Nov23 (#1857)
MicrosoftDocs/WSL: WSL/about.md

---
title: What is Windows Subsystem for Linux 
description: Learn about the Windows Subsystem for Linux, including the different versions and ways you can use them. Microsoft Loves Linux.
ms.date: 11/20/2023
ms.topic: article
---

# What is the Windows Subsystem for Linux?

> Windows Subsystem for Linux (WSL) is a feature of Windows that allows you to run a Linux environment on your Windows machine, without the need for a separate virtual machine or dual booting.  WSL is designed to provide a seamless and productive experience for developers who want to use both Windows and Linux at the same time.

Reviewer: ```suggestion
Windows Subsystem for Linux (WSL) is a feature of Windows that allows you to run a heavily integrated Linux environment on your Windows machine.  WSL is designed to provide a seamless and productive experience for developers who want to use both Windows and Linux at the same time.
```
Adjusted wording since we do use a virtual machine for WSL
